.gitlab/ci/on-push.yml

```diff
@@ -129,5 +141,9 @@ server_master:
   extends:
     - .test_content_on_server_instances_base
     - .nightly-rule
+    - .notify-slack
```

Comment:
```suggestion
```
Can remove it from here, and have `.test_content_on_server_instances_base` extend from `.default-after-script`, and since `server_master` is the only job with the `SLACK_ONLY_IF` environment variable defined, it'll be the only one that will send a slack message.